tests/js/spec/components/forms/selectCreatableField.spec.jsx

```diff
@@ -18,7 +18,7 @@ describe('SelectCreatableField', function () {
 
     // Click on create option
     openMenu(wrapper, {control: true});
-    wrapper.find('SelectControl Option div').simulate('click');
+    wrapper.find('SelectControl Option Label').simulate('click');
```

Comment:
Since there are multiple divs inside the replacement `<Option />`, we need to be more specific and refer to the element that wraps around the label text.